### ISSUE TITLE
Bundeslogo text and selected option not visible in IE with windows High Contrast Mode

### ISSUE THREAD

**Issue by angelogti** [swiss/styleguide#634]
When using windows high contrast mode on IE, the text of the logo and the selected option of dropdowns are not visible
![image](https://user-images.githubusercontent.com/34123685/34713299-1e817072-f526-11e7-903b-35ea64f62107.png)

![image](https://user-images.githubusercontent.com/34123685/34713404-72f7a9d2-f526-11e7-8845-01784fcc0865.png)

The problem is only on IE. High contrast mode has no effect on what the other browsers show. When testing on Chrome with an extension "HoherKontrast", everything works as expected. 

![image](https://user-images.githubusercontent.com/34123685/34713718-56d62458-f527-11e7-9fa9-0d567629f7c6.png)

Can we do anything or is it a IE bug?

**Comment by gillerr** [comment]
Won't fix
